src/uct/ib/base/ib_md.c

```diff
@@ -309,13 +310,40 @@ uint8_t uct_ib_md_get_atomic_mr_id(uct_ib_md_t *md)
 #endif
 }
 
+static void uct_ib_md_print_mem_reg_err_msg(ucs_log_level_t level, void *address,
+                                            size_t length, uint64_t exp_access,
+                                            int exp)
+{
+    char *str = ucs_alloca(32);
+    char msg[400] = {0};
+
+    if (exp) {
```

Comment:
instead of "exp" can pass string (`const char *exp_prefix`) which is either "exp_" or "", and use it in snprintf